src/js/components/help/components/pages/getting-started/GettingStarted.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import styled from 'styled-components'
+//import image from '../../../../../img/help-pages/help login page desktop.jpg'
+import image from '../../../../../../img/help-pages/help login page desktop.jpg'
+import { Link} from 'react-router-dom'
+import Topic from '../../topic'
+import Description from '../../help-description'
+import Sidebar from '../../help-topics'
+
+const StyledGettingStarted =styled.div`
+
+.help-description-container{
+  display:flex;
+  margin 5px;
+  padding: 5px;
+}
+
+img{
+width:700px;
+height: 500px;
+}
+
+.help-main-content{
+  font-size: 15px;
+  margin-left:80px;
+  max-width: 650px;
+}
```

Comment:
Noticed this same styles in the previous PR on the general help pages. We really should abstract this into a component on its own.